content/docs/06-integrations/00-aws-autoscaler.md

```diff
@@ -14,81 +14,174 @@ import InfoBox from 'shared/components/InfoBox';
+<br />
+
+  ```yaml
+  managedMachinePool:
+    #roleName: {{ name of the self-managed role | format "${string}" }}
+    ## A list of additional policies to attach to the node group role
+    roleAdditionalPolicies:
+    - {{ arn for the policy1 | format "${string}" }}
+  ```
+
+
+## Usage
+
+The first step is to define an IAM policy in the AWS account associated with Palette. Then, switch to Palette to update the Kubernetes layer manifest and add the AWS Cluster Autoscaler pack. Finally, deploy your cluster and scale down the instance type to validate that AWS Cluster Autoscaler works correctly. The following sub-sections will provide more detailed instructions.
+<br />
+
+### Define IAM Policy
+Define the new IAM policy, using the full Cluster Autoscaler policy outlined in the prerequisites section above, and give it a name, for example, *PaletteEKSClusterAutoscaler*. After defining the IAM policy, copy its ARN to use in the next step. For example, the policy ARN will be similar to `arn:aws:iam::650628870702:policy/PaletteEKSClusterAutoscaler`. 
+<br />
+
+### Update the Kubernetes Layer Manifest 
+Log in to Palette, and create a new cluster profile for AWS EKS or update an existing one. In the Kubernetes layer of your cluster profile, update the `managedMachinePool.roleAdditionalPolicies` section in the Kubernetes pack's manifest with the newly created IAM policy ARN. The snapshot below shows the section to update with the policy ARN. 
+
+
+![Snapshot showing the ARN  added to the Kubernetes pack's manifest.](/integrations_aws-cluster-autoscaler_k8s-manifest.png)
+
+
+The code block below shows the updated `managedMachinePool.roleAdditionalPolicies` section with a sample policy ARN, `"arn:aws:iam::650628870702:policy/PaletteEKSClusterAutoscaler"`, as an example. Before you use the following code block, replace the ARN below with yours. 
+<br />
+
+  ```yaml
+  managedMachinePool:
+    # roleName: {{ name of the self-managed role | format "${string}" }}
+    # A list of additional policies to attach to the node group role
+    roleAdditionalPolicies:
+    - "arn:aws:iam::650628870702:policy/PaletteEKSClusterAutoscaler"
+  ```
+  <br />
+
+<InfoBox>
+
+If you do not want to update the Kubernetes pack's manifest, you can add an *inline* IAM policy to the cluster's node group post deployment. Refer to this [AWS guide](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_manage-attach-detach.html#add-policies-console) on how to embed an inline policy for a user or role.
+
+</InfoBox>
+
+
+Note that when you deploy an EKS cluster using Palette, Palette creates two IAM roles in the AWS account automatically - one for the cluster and another for the node group. Adding the ARN of your newly created IAM policy to the Kubernetes pack's manifest will make Palette add that IAM policy to the node group during deployment. 

```

Comment:
```suggestion
When you deploy an EKS cluster using Palette, Palette automatically creates two IAM roles in the AWS account. One for the cluster and another for the cluster's node group. Adding the ARN of your newly created IAM policy to the Kubernetes pack's manifest will make Palette add that IAM policy to the node group during deployment. 
```

Can we share the name of the roles, or the name prefix in case they have dynamic values at the end.